pkg/playground/react-demo-listing.jsx

```diff
@@ -51,6 +51,13 @@
             window.alert("navigated to item: " + msg);
         };
 
+        var handleAddClick = function(event) {
+            if (event.button !== 0)
+                return;
+
+            window.alert('this link could add a row');
```

Comment:
We don't have a pattern here yet. We're showing a dialog in the one place we're going to use this ("Get new image" on the containers page), but I can see this adding a template row for simpler lists as well.

That said, I don't mind changing the text to avoid confusion.